src/components/headers_overlay/headers_overlay.ts

```diff
@@ -344,6 +345,7 @@ css/* scss */ `
       width: fit-content;
       gap: 5px;
       transform: translate(-50%, 0);
+      top: ${HEADER_HEIGHT / 2 - ICON_EDGE_LENGTH / 2 + "px"};
```

Comment:
Uh ? What's this offset for ? The icons are now not centered

Reply:
I meant that after your fix, the arrows are not centered vertically inside the headers. After your fix : 
![image](https://github.com/odoo/o-spreadsheet/assets/31045223/130d1207-c9af-428e-a8b3-4c80b606ef20)

Without the 4px top offset that you added. The arrows are now correctly centered vertically
![image](https://github.com/odoo/o-spreadsheet/assets/31045223/924ce6f2-77bd-4646-8617-949b1b88220a)

Reply:
Oh, you want it centered?

I initially thought it should be positioned slightly below so that both the character and icon are visible if the column size is smaller.

I've made that adjustment. Thanks!